src/controllers/EventsController.js

```diff
@@ -39,7 +40,7 @@ class EventsController extends Controller {
       this.response.write(`event: ${event.name}\n`);
       this.response.write(`data: ${JSON.stringify(eventToApi(event))}\n\n`);
     } catch (error) {
-      logger.error(`pipeEvents - write error: ${error}`);
+      logger.error({ err: error }, 'pipeEvents - write error');
```

Comment:
This is a feature from Bunyan called Seriallizers (you can also extend these for your own objects). The standard for the Node Error object is err: . So the error is handled and printed correctly including stack etc. .  I would suggest to use Bunyan standard, if we use Bunyan.
Later I would suggest to make own Seriallizers for internal Spark Objects, to make the log better